Lesson15/Lesson15-cansel/script.js

```diff
@@ -0,0 +1,200 @@
+// CODE HERE
+const points = 10;
+points += 5;
+console.log('Ex.15.', points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+// CODE HERE
+const name = "Alice";
+const age = 30;
+const city = "Paris";
+const message = `${name} (${age}) lives in ${city}`;
+console.log('Ex.16.', message);
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+// CODE HERE
+const x = 5;
+const y = 10;
+const z = 15;
+const totalSum = x + y + z;
+console.log('Ex.17.', totalSum);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+// CODE HERE
+const dividend = 10;
+const divisor = 3;
+const divisionResult = dividend / divisor;
+const differenceResult = dividend - divisor;
+const remainderResult = dividend % divisor;
+console.log(`'Ex.18.1'${divisionResult}`);
+console.log(`'Ex.18.2'${differenceResult}`);
+console.log(`'Ex.18.3'${remainderResult}`);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+// CODE HERE
+const firstName = "Cansel";
+const lastName = "Yilmaz Canturk";
+const fullName = `${firstName} ${lastName}`;
+console.log('Ex.19.', fullName);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+// CODE HERE
+const firstFactor = 7;
+const secondFactor = 2;
+const product = firstFactor * secondFactor;
+console.log('Ex.20.', product);
+
+// 21. Log the value of Math.PI.
+// CODE HERE
+const piValue = Math.PI;
+console.log('Ex.21.', piValue);
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+// CODE HERE
+const counter = 0;
+console.log(`'Ex.22.1'${counter}`);
+counter++;
+console.log(`'Ex.22.2'${counter}`);
+counter += 1;
+console.log(`'Ex.22.3'${counter}`);
+counter = counter + 1;
+console.log(`'Ex.22.4'${counter}`);
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+// CODE HERE
+const initialTemperature = 20;
+initialTemperature += 5;
+console.log('Ex.23.', initialTemperature);
+
+// 24. Declare numberEx9=6. Increment it using the prefix ++ operator and log both the variable and the incremented value.
+// CODE HERE
+const numberEx9 = 6;
+console.log(`'Ex.24.1'${numberEx9}`);
+const incrementedValue = ++numberEx9;
+console.log(`'Ex.24.2'${incrementedValue}`);
+console.log(`'Ex.24.3'${numberEx9}`);
+
+// 25. Declare numberEx10=8. Increment it using the postfix ++ operator and log both the original variable and the incremented value.
+// CODE HERE
+const numberEx10 = 8;
+console.log(`'Ex.25.1'${numberEx10}`);
+const returnedValue = numberEx10++;
+console.log(`'Ex.25.2'${returnedValue}`);
+console.log(`'Ex.25.3'${numberEx10}`);
+
+// 26. Declare numberEx11=-3. Increment it by 1 using prefix ++, then multiply the result by 2 and log the final value.
+// CODE HERE
+const numberEx11 = -3;
```

Comment:
This variable should be defined as 'let' so it's possible to increment it later.